engines/native/system/user_windows.go

```diff
@@ -0,0 +1,21 @@
+package system
+
+// User is a representation of a system user account.
+type User struct {
+	name       string
+	homeFolder string
+}
+
+// CreateUser will create a new user, with the given homeFolder, set the user
+// owner of the homeFolder, and assign the user membership of given groups.
+func CreateUser(homeFolder string, groups []*Group) (*User, error) {
+	panic("Not implemented")
```

Comment:
Note: it really was my hope that it would be simple to implement the system package on Windows and OS X... So we can quickly do it later.